docs/integrations/sources/hubspot.md

```diff
@@ -73,6 +73,7 @@ This connector supports only authentication with API Key. To obtain API key for
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.15   | 2021-09-?? | [????](https://github.com/airbytehq/airbyte/pull/????) | Use correct schema for `owners` stream |
```

Comment:
Please update ????

Reply:
I'll update this at the version bumping stage before the connector is released, along with the release date.